method/broadcast.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$comment": "Previously called message.json, I'd suggest to rename it to broadcast as 1) it's what it does 2) it disambiguates it from the mid-level field message",
+    "$comment": "If you agree, we would also need to change the enum in generic from message to broadcast",
+    "$schema": "http://json-schema.org/draft/2019-09/schema#",
+    "$id": "http://pop2020.dedis.ch/low-level/broadcast.json",
```

Comment:
How is this id generated?

Reply:
This is the `$id` "template" that @pierluca used in the outdated draft

According to the [JsonSchema doc](https://json-schema.org/understanding-json-schema/basics.html#declaring-a-unique-identifier), the `$id` should be set to 
>a URL at a domain you control

Could we maybe use the github URL? Maybe consider that all schemas are already on the `proto-specs` branch (thus `"$id": "https://github.com/dedis/student20_pop/tree/proto-specs/<path-to-file>"`)

Reply:
I guess it is fine to keep using the dedis.ch. We control this domain name.